apix-integrationtests-server/src/main/java/eu/xenit/apix/util/SolrTestHelperImpl.java

```diff
@@ -35,8 +35,7 @@ public class SolrTestHelperImpl implements SolrTestHelper {
     public SolrTestHelperImpl(
             @Qualifier("global-properties") Properties globalProperties,
             @Qualifier("Search") SwitchableApplicationContextFactory searchSubSystem,
-
-            AlfrescoServerInfo alfrescoServerInfoParam) {
+            AlfrescoServerInfo alfrescoServerInfoParam) { // @Qualifier("defaultDataSource") ??
```

Comment:
i don't think you need this qualifier (or the comment)

Reply:
Was used in old code, left it in case i needed to specify.
Removed.